devices/apa102/apa102_test.go

```diff
@@ -636,15 +636,15 @@ func TestInit(t *testing.T) {
 
 //
 
-func BenchmarkWriteWhite(b *testing.B) {
-	b.ReportAllocs()
-	pixels := [150 * 3]byte{}
-	for i := range pixels {
-		pixels[i] = 0xFF
+type fn func(int) byte
```

Comment:
I would have expected a triplet, e.g.

    type genColor func(int) [3]byte

then

    pixel = append(pixels, f(i)...)

but I don't mind that much.

Reply:
Done.